config/initializers/slack.rb

```diff
@@ -0,0 +1,3 @@
+Slack.configure do |config|
+  config.token = ENV['SLACK_API_TOKEN']
```

Comment:
This project declares [and defaults] ENV vars in [this initializer](https://github.com/artsy/horizon/blob/master/config/initializers/_config.rb) by convention, so this should be added there, then referenced here. This eases stubbing for tests and keeps available configuration visible in a single location.

More generally, though, Horizon tries to support different orgs and projects using different settings, so I wonder if it's even appropriate to have a global Slack token configured. Instead, this would make a good addition to the `Profile#environment` that each deploy strategies already reference to get the necessary authorization. Similarly, `DeployStrategy#arguments` is a json field that could offer a new [optional] `slack_channel` property indicating which channel to notify. This field is already being used above to support custom branch setups.

Reply:
If we were truly wanting to support multiple organizations, then in my mind we should likely go for the whole oauth flow and enable multi-slack organization usage. That's a bit of extra work though and we're trying to be pretty scrappy on this being as it's a side thing. 

Being as @ansor4 implemented your suggestion, maybe that's a good hold-over until we can do the full oauth setup.